src/formulations/inverse-dynamics-formulation-acc-force.cpp

```diff
@@ -437,8 +444,17 @@ bool InverseDynamicsFormulationAccForce::decodeSolution(const HQPOutput & sol)
 {
   if(m_solutionDecoded)
     return true;
+
+  // Add all measured external forces to dynamic model
+  Vector h_fext;
+  h_fext.setZero(m_v);
+  for (auto& it : m_measuredForces)
+  {
+    h_fext += it->measuredForce.computeJointTorques(m_data);
+  }
```

Comment:
This quantity has already been computed above, so it should be stored in a member variable and re-used here.

Reply:
It has already been computed but in a different method, I moved the variable into .hpp in my last commit but I think it still needs to be computed here, or maybe I miss something ?

Reply:
Ok so I may have an issue here :  if I remove this 'for loop' and use h_fext value below, nothing works, I have no errors but my virtual human is doing is making a mess (should be static) So do I need to set any specific declaration of this variable in the constructor ?

Reply:
No, I don't see why it shouldn't work. If you push the code here I can take a look though

Reply:
It's good, it was coming from a missing set of the h_fext vector size